drivers/video/gc2145.c

```diff
@@ -689,22 +689,25 @@ struct gc2145_data {
 	struct video_format fmt;
 };
 
-#define GC2145_VIDEO_FORMAT_CAP(width, height, format) \
-	{                                              \
-	.pixelformat = format,                         \
-	.width_min = width,                            \
-	.width_max = width,                            \
-	.height_min = height,                          \
-	.height_max = height,                          \
-	.width_step = 0,                               \
-	.height_step = 0,                              \
+#define GC2145_VIDEO_FORMAT_CAP(width, height, format)                                             \
+	{                                                                                          \
+		.pixelformat = format, .width_min = width, .width_max = width,                     \
+		.height_min = height, .height_max = height, .width_step = 0, .height_step = 0,     \
 	}
 
 static const struct video_format_cap fmts[] = {
+	GC2145_VIDEO_FORMAT_CAP(320, 240, VIDEO_PIX_FMT_RGB565),   /* QVGA  */
+	GC2145_VIDEO_FORMAT_CAP(640, 480, VIDEO_PIX_FMT_RGB565),   /* VGA  */
 	GC2145_VIDEO_FORMAT_CAP(1600, 1200, VIDEO_PIX_FMT_RGB565), /* UXGA  */
 	{0},
 };
 
+enum vga_resolutions {
```

Comment:
Is there a sensor-specific reason for `vga_` here? Can this (and the variables below) be simply `resolution`? 
Also, if `fmts` is linked with this enum, you could define the enum before the const array and explicitly link the index with it.